src/validator/MutateValidator.cpp

```diff
@@ -70,15 +61,23 @@ Status InsertVerticesValidator::check() {
         }
 
         std::vector<std::string> names;
-        auto props = item->properties();
-        // Check prop name is in schema
-        for (auto *it : props) {
-            if (schema->getFieldIndex(*it) < 0) {
-                LOG(ERROR) << "Unknown column `" << *it << "' in schema";
-                return Status::SemanticError("Unknown column `%s' in schema", it->c_str());
+        if (item->isDefaultPropNames()) {
+            propSize_ = schema->getNumFields();
+            for (size_t i = 0; i < propSize_; ++i) {
+                const char* propName = schema->getFieldName(i);
+                names.emplace_back(propName);
```

Comment:
the storage supported the request without put the propNames, if empty, the storage will use the schema props.